client/history/client.go

```diff
@@ -821,7 +821,10 @@ func (c *clientImpl) GetReplicationMessages(
 			requestContext, responseInfo := rpc.ContextWithResponseInfo(requestContext)
 			resp, err := c.client.GetReplicationMessages(requestContext, req, append(opts, yarpc.WithShardKey(peer))...)
 			if err != nil {
-				c.logger.Warn("Failed to get replication tasks from client", tag.Error(err))
+				c.logger.Warn("Failed to get replication tasks from client",
+					tag.Error(err),
+					tag.ShardReplicationToken(req),
```

Comment:
huh.  what's the token / is this something that can be correlated with other logs or something?